pom.xml

```diff
@@ -101,9 +104,12 @@
         <version.org.apache.geronimo.specs.geronimo-validation_1.0_spec>1.1</version.org.apache.geronimo.specs.geronimo-validation_1.0_spec>
         <version.org.apache.myfaces.core.myfaces-bundle>2.1.15</version.org.apache.myfaces.core.myfaces-bundle>
         <version.commons-beanutils>1.9.2</version.commons-beanutils>
-        <version.org.ops4j.pax.cdi>0.11.0.redhat-001</version.org.ops4j.pax.cdi>
+        <pax.cdi.feature.version>1.0.0.RC1-redhat-001</pax.cdi.feature.version>
```

Comment:
Is there a specific reason to not follow the `version.<group-id>` pattern for the property names?